.github/scripts/query_observe.py

```diff
@@ -0,0 +1,299 @@
+import configparser
+import json
+import re
+import logging
+import requests
+import os, sys
+import datetime
+from datetime import timezone
+
+
+# Create a logger instance
+def setup_logger(log_level, log_format):
+    """
+    Setup logger with specified log level and format.
+
+    @param log_level: Logging level (default is DEBUG)
+    @param log_format: Logging format (default is '%(asctime)s - %(levelname)s - %(message)s')
+    """
+    logger.setLevel(log_level)
+    formatter = logging.Formatter(log_format)
+
+    # Create console handler and set level to debug
+    ch = logging.StreamHandler()
+    ch.setLevel(log_level)
+    ch.setFormatter(formatter)
+
+    # Add console handler to logger
+    logger.addHandler(ch)
+
+
+def get_bearer_token() -> str:
```

Comment:
Not for ths pr but we use this over an over we should have a common library and make some classes we can import so we stop repeating same code all over

Reply:
I agree. At least the below:
- Bearer Token
- Send Observe Query

Reply:
https://observe.atlassian.net/browse/OB-31450 
For future